client/src/components/userprofiles/UserProfilesList.jsx

```diff
@@ -1,11 +1,13 @@
 import { useEffect, useState } from "react";
 import { changeIsActiveStatus, getProfiles } from "../../managers/userProfileManager";
 import { Button, ButtonToolbar, Card, CardBody, CardTitle } from "reactstrap";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 
 export default function UserProfileList() {
   const [userprofiles, setUserProfiles] = useState([]);
 
+  const navigate = useNavigate();
+
   const getUserProfiles = () => {
     getProfiles().then(setUserProfiles);
   };
```

Comment:
Add a newline between the closing tag parenthesis of `getUserProfiles()` and the useEffect below it